.changeset/fuzzy-years-taste.md

```diff
@@ -0,0 +1,37 @@
+---
+'@clerk/backend': major
+'@clerk/nextjs': major
+'@clerk/remix': major
+'gatsby-plugin-clerk': major
+---
+
+## Breaking Changes
+
+### @clerk/backend
+
+Replace `Clerk` with `createClerkClient` [SDK-1058]
+
+
+```js
+import { Clerk } from '@clerk/backend';
+const clerk = Clerk({ secretKey: '...' });
+```
+
+now becomes:
+
+```js
+import { createClerkClient } from '@clerk/backend';
+const clerk = createClerkClient({ secretKey: '...' });
+```
+
+### @clerk/gatsby-plugin-clerk
+
+No longer returning `Clerk`. Please replace with `createClerkClient`
+
+### @clerk/nextjs
+
+No longer returning `Clerk`. Please replace with `createClerkClient`
+
+### @clerk/remix
```

Comment:
At least in https://clerk.com/docs/references/backend/overview we already told people to use `createClerkClient`, so this message needs some nuance that people probably don't need to change anything if they followed the docs recently.